roles/cloudflared/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Cloudflared Docker Container
+  docker_container:
+    name: cloudflared
+    image: cloudflare/cloudflared
+    pull: true
+    env:
+      TZ: "{{ ansible_nas_timezone }}"
+    restart_policy: unless-stopped
+    command: "tunnel --no-autoupdate run --token {{ cloudflared_token }}"
+    network_mode: "host"
+    recreate: true
```

Comment:
Is it necessary to recreate the container on every run?

Reply:
Sorry my mistake, it's left behind from my personal setup in which I was setting watchtower labels for autoupdate, which requires recreating the container